toolset/setup/linux/languages/php7.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+RETCODE=$(fw_exists ${IROOT}/php.installed)
```

Comment:
This will be problematic on our production system where php and php7 need to coexist. Specifically, this should probably be `php7.installed` throughout.